CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ project(ignition-gui3 VERSION 3.10.0)
 #============================================================================
 # Find ignition-cmake
 #============================================================================
-find_package(ignition-cmake2 2.8.0 REQUIRED)
+find_package(ignition-cmake2 2.13 REQUIRED)
```

Comment:
That isn't strictly necessary here because there isn't a coverage file, but it's nice for whoever is checking coverage locally, because they will get some other things ignored.